src/moustachu.nim

```diff
@@ -190,14 +195,14 @@ proc render(tmplate: string, contextStack: seq[Context]): string =
 
   result = join(renderings, "")
 
-proc render*(tmplate: string, c: Context): string =
+proc render*(tmplate: string, c: Context, pwd="."): string =
   var contextStack = @[c]
-  result = tmplate.render(contextStack)
+  result = tmplate.render(contextStack, pwd)
 
 
 when isMainModule:
   import json
-
+  import os
```

Comment:
Given that `os` is imported at the top of the file, we shouldn't have to import it again.